build/ie.min.css

```diff
@@ -1 +0,0 @@
-main{overflow-x:hidden}
```

Comment:
What was this for?

Reply:
I have no idea, this is generated in the build process. It doesn't seem related to any of my SASS changes. I could rerun the build process maybe.